modules/xmpp/SdpConsistency.js

```diff
@@ -78,6 +38,9 @@ export default class SdpConsistency {
      *  in future calls to makeVideoPrimarySsrcsConsistent
      */
     setPrimarySsrc (primarySsrc) {
+        if (typeof primarySsrc !== 'number') {
```

Comment:
maybe simplify by assigning the return value of `parseInt` ? That would take care of raising the appropriate exception.